packages/@aws-cdk/aws-kinesisfirehose-destinations-alpha/lib/private/helpers.ts

```diff
@@ -87,8 +87,8 @@ export function createBufferingHints(
   }
 
   const intervalInSeconds = interval?.toSeconds() ?? 300;
-  if (intervalInSeconds < 60 || intervalInSeconds > 900) {
-    throw new Error(`Buffering interval must be between 60 and 900 seconds. Buffering interval provided was ${intervalInSeconds} seconds.`);
+  if (intervalInSeconds > 900) {
```

Comment:
```suggestion
  if (intervalInSeconds < 0 || intervalInSeconds > 900) {
```
Let's keep the lower bound validation.

Reply:
`interval` is a `cdk.Duration` and `cdk.Duration` does not accept negative numbers.
Initially, I wanted to keep the lower bound and when trying to test that, `cdk.Duration` threw an exception about not allowing negative numbers. It was then I decided to drop the lower bound as it's enforced by `cdk.Duration`.

Reply:
Wonderful!